content/docs/command-reference/add.md

```diff
@@ -378,3 +380,49 @@ system that can handle it), they can use `dvc pull` as usual:
 A       data.xml
 1 file added and 1 file fetched
 ```
+
+## Example: Transfer to the cache
+
+When you have a large dataset in an external location, you may want to add it to
+your cache without actually copying it into the workspace first. This might be
+due to cache and workspace are in separate disks, which only the cache can
+handle that size of data. After the data is saved to your cache, we link it to
+your workspace with the
+[preffered links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache).
+
+Let's initalize a DVC project;
+
+```dvc
+$ mkdir example # workspace
+$ cd example
+$ git init
+$ dvc init
+```
+
+Afterwards, let's setup a shared cache by following
+[this](https://dvc.org/doc/use-cases/shared-development-server#preparation)
+tutorial. When it is ready to go, we can add `data.xml` to our cache directly;
+
+```
+$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml
+```
+
+Depending on the cache type configured on our workspace (can be set using
+`cache.type` config value), the data is either linked up or just copied over.
+For this use case, a reflink or a symlink is suggested.
+
+```
+$ ls
+data.xml data.xml.dvc
+```
```

Comment:
I think we can merge this code block to the previous one, and skip this paragraph because we have to explain the linking earlier (see previous comments).

> BTW prob. no need to mention "just copied over": That doesn't seem useful in the tp-cache scenario 🙂